modules/distribution/src/sso/views/register_new_user.jag

```diff
@@ -0,0 +1,200 @@
+                                    var claims = claimsModule.getDefaultClaims();
+                                    var renderedClaims = {};
+
+                                    var claimUriOverrides = {
+                                        USERNAME: {
+                                            render: false // because this is already rendered by default
+                                        },
+                                        PASSWORD: {
+                                            render: false // because this is already rendered by default
+                                        },
+                                        EMAILADDRESS: {
+                                            render: true,
+                                            customClass: 'email'
+                                        }
+                                    };
+
+                                    for (var claimIndex = 0; claimIndex < claims.length; claimIndex++) {
+                                        var override = claimUriOverrides[claims[claimIndex].claimUri.toUpperCase()];
+                                        // render claim if it's required and not overriden to --> render:false
+                                        if (claims[claimIndex].isRequired && (!override || override.render !== false)) {
+                                            var formElementName = 'reg-' + claims[claimIndex].displayTag.toLowerCase().replace(' ', '-');
+                                            var validationClass = "";
+                                            var span = "";
+                                            if(claims[claimIndex].displayTag == "Email"){
+                                                validationClass = "email";
+                                                span = "col-xs-12 col-sm-12 col-md-12 col-lg-12";
+                                            }else{
+                                                span = "col-xs-12 col-sm-12 col-md-6 col-lg-6"
+                                            }
+                                            %>
+                                        <div class="<%=span%> form-group">
+                                            <label><%=claims[claimIndex].displayTag%></label>
+                                            <input type="text" name="<%=formElementName%>" data-claim-uri="<%=claims[claimIndex].claimUri%>" class="form-control <%=validationClass%> required <%=claims[claimIndex].customClass%>">
+                                        </div>
+                                    <%
+                                            renderedClaims[formElementName] = claims[claimIndex].claimUri;
+                                        }
+                                    }
+                                    %>
+
+                                    <input type="hidden" name="reg-claims" value='<%=renderedClaims%>'
```

Comment:
This is html attribute value, hence agreed to change it to double quotes. But unable to fix as this code is obsolete and no longer available